Pipfile

```diff
@@ -77,4 +78,3 @@ python_version = "3.6"
 test = "python setup.py test"
 build_sphinx = "python setup.py build_sphinx"
 dev = "pytest --no-cov -vs tests"
-setup = "./scripts/setup"
```

Comment:
If you delete `setup` you should change `./scripts/setup` when it asks user to use `pipenv run setup` and suggest `pipenv run ./scripts/setup` instead.
Same thing in `INSTALL.rst` file.